Pipfile

```diff
@@ -39,6 +39,7 @@ django-prometheus = "~=2.3.1"
 django-admin-autocomplete-filter = "~=0.7.1"
 django-pgtrigger = "~=4.11.1"
 logging-utilities = "~=4.4.1"
+stac-api-validator = "*"
```

Comment:
Should not this be in the dev-package ? In the package section we should have fix major, minor version.

Reply:
:+1: Yes that makes sense. I added to dev-deps and fixed the version to the current latest.